index.js

```diff
@@ -63,7 +62,10 @@ Region.prototype.getChunk = function(x, z) {
     this.dataView.seek(offset)
     length = this.dataView.getInt32()
     version = this.dataView.getUint8()
-    data = new Uint8Array(this.buffer, this.dataView.tell(), length)
```

Comment:
I was just trying to track down this same issue and I ended up using `data = this.buffer.slice(this.dataView.tell(), this.dataView.tell() + length);` in place of this line. That code can certainly be improved, but it ran successfully in node 0.10.4, 0.12.9, and 5.4.1